docs/usage/self-hosted-configuration.md

```diff
@@ -667,6 +667,50 @@ By default, Renovate does not autodiscover repositories that are mirrors.
 
 Change this setting to `true` to include repositories that are mirrors as Renovate targets.
 
+## inheritConfig
+
+When you enable this option, Renovate will look for the `inheritConfigFileName` file (defaults to `org-inherited-config.json`) in the `inheritConfigRepoName` repository (defaults to `{{parentOrg}}/renovate-config`) before processing a repository, and read this in as config.
+If the repository is in a nested organization/group on a supported platform such as GitLab, such as `topGroup/nestedGroup/projectName` then Renovate will look in `topGroup/nestedGroup/renovate-config`.
+
+If `inheritConfig` is `true` but the inherited config file does _not_ exist then Renovate will proceed without warning.
+If the file exists but cannot be parsed, then a config warning issue will be raised and the job aborted.
+
+The inherited config may include all valid repo config and these config options:
+
+- `bbUseDevelopmentBranch`
+- `onboarding`
+- `onboardingBranch`
+- `onboardingCommitMessage`
+- `onboardingConfig`
+- `onboardingConfigFileName`
+- `onboardingNoDeps`
+- `onboardingPrTitle`
+- `onboardingRebaseCheckbox`
+- `requireConfig`
+
+This means that organizations can change/control default behavior like whether configs are required and how repositories are onboarded.
+
+We disabled `inheritConfig` in the Mend Renovate App to avoid millions of wasted API calls per week.
+This is because each `404` from the GitHub API for a missing org inherited config counts as a used API call.
+A smart/dynamic approach will be added in future so that it can be enabled selectively per-org.
```

Comment:
```suggestion
We will add a smart/dynamic approach in future, so that we can selectively enable `inheritConfig` per organization.
```

If you want to commit to this work, we can have a stronger sentence like this one.

`@secustor` mentioned creating a tracking issue for this work, so people can subscribe to that issue. So if you want you can create the issue, and put it in the suggestion.